pkg/virt-launcher/virtwrap/network/podinterface_test.go

```diff
@@ -378,7 +384,7 @@ var _ = Describe("Pod Network", func() {
 				for _, proto := range ipProtocols() {
 					mockNetwork.EXPECT().HasNatIptables(proto).Return(true).Times(2)
 				}
-				mockNetwork.EXPECT().IsIpv6Enabled().Return(true).Times(3)
+				Expect(CheckIsIpv6Enabled()).To(BeTrue())
```

Comment:
Why do you need to check the value of a stub method? If it possible I would keep the 'Times' check.